package.json

```diff
@@ -15,7 +15,6 @@
     "@react-native-community/push-notification-ios": "^1.2.2",
     "@react-navigation/native": "^5.9.4",
     "@react-navigation/stack": "^5.14.5",
-    "@t2tx/react-native-universal-pedometer": "https://github.com/francisli/react-native-universal-pedometer#b50b4766de3bc487147d7c314fe36abc7763e730",
```

Comment:
This package is used by the iOS app for the pedometer... was it required for you to remove it in order to get the build to succeed?

Reply:
I didn't manually remove this package, so I'm assuming it happened from my `fastlane android beta` command. I couldn't tell you why it didn't push everything up, but it did make a local commit. That commit had some extra garbage so I cleaned it up.

Reply:
If you think it'd be better, I could update the PR to just be `build.gradle`.

Reply:
Yes, let's revert the package.json and package-lock.json changes...

Reply:
Done